### PR TITLE
Diamond fix memory stats

### PR DESCRIPTION
**Pull request by sbuliarca**
App pull request https://github.com/Financial-Times/coco-diamond/pull/1/files

### CODE REVIEW

File: diamond.service [hunk 1]
```diff
@@ -8,19 +8,20 @@ Environment="DOCKER_APP_VERSION=latest"
 TimeoutStartSec=0
 ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p_)" > /dev/null 2>&1'
 ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p_)" > /dev/null 2>&1'
-ExecStartPre=/bin/bash -c 'docker history coco/coco-diamond:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/coco-diamond:$DOCKER_APP_VERSION'
+ExecStartPre=/bin/bash -c 'docker history coco/coco-diamond-auto:$DOCKER_APP_VERSION >/dev/null 2>&1 || docker pull coco/coco-diamond-auto:$DOCKER_APP_VERSION'
 
 ExecStart=/bin/bash -c '\
   ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
-  IFS=. read -r -a HOSTNAME_SLICED <<< "$HOSTNAME"; \
```
Comment (sbuliarca) [inline diamond.service]: Slicing is not needed any more, as it is done in the wrapper needed for Kubernetes. App pull request https://github.com/Financial-Times/coco-diamond/pull/1/files

File: diamond.service [hunk 2]
```diff
@@ -8,19 +8,20 @@ Environment="DOCKER_APP_VERSION=latest"
...
   -e GRAPHITE_PORT=2003 \
-  -e DOCKER_HOSTNAME=$ENV.$HOSTNAME_SLICED \
+  -e PREFIX=coco.servers \
+  -e NODE_NAME="$HOSTNAME" \
+  -e ENVIRONMENT=$ENV \
   -e INTERVAL=60 \
   -v /proc:/host_proc:ro \
   -v /var/run/docker.sock:/var/run/docker.sock:ro \
-  coco/coco-diamond:$DOCKER_APP_VERSION'
+  coco/coco-diamond-auto:$DOCKER_APP_VERSION'
```
Comment (sbuliarca) [inline diamond.service]: I created a new DH repository, as the old one did not have auto build.

**Comment by sbuliarca** [comment]
The application pull request is here: https://github.com/Financial-Times/coco-diamond/pull/1/files

**Comment by davidbalazs93** [comment]
please remove the old Docker hub repo after deploying this new version of diamon service

**Pull request merged**